src/ansible/roles/packages/tasks/RedHat.yml

```diff
@@ -1,6 +1,173 @@
-- name: 'Packages are the same as in CentOS {{ ansible_distribution_major_version }}'
-  include_tasks: '{{ include_centos }}'
-  loop_control:
-    loop_var: include_centos
-  with_first_found:
-  - files: '{{ "CentOS" | distro_includes(ansible_distribution_major_version) }}'
+- name: Install packages for ground base image
```

Comment:
This change is not needed. Packages can be perfectly shared with CentOS.

Reply:
You clearly did not spot that for RedHat I am installing ipa that is used for testing in our tests instead of freeipa. There are also some debuginfo packages not available on RHEL so I am ignoring them for now (so the lack is visible and can be revisited later).